app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = list()
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.__class__.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.__dict__['name']}, "
+                + f"Health: {self.__dict__['health']}, "
+                + f"Hidden: {self.__dict__['hidden']}}}")
```

Comment:
Use interpolation instead of concatenation will be better. You can find this advice in checklist.md